JSDemos/Demos/Form/CustomizeItem/description.md

```diff
@@ -1 +1,5 @@
-The Form component can create form items automatically for each field of&nbsp;the **formData** object. If&nbsp;you&rsquo;d prefer to&nbsp;configure form items manually, pass the required array of&nbsp;configuration objects to&nbsp;the **items** property. You can specify the name of&nbsp;the bound data field, editor type, editor configuration, validation properties (etc.) for each form item.
\ No newline at end of file
+You can use the [items[]](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#items) array to configure all form items. This array can contain strings ([formData](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#formData) field names) and objects (item configurations). Use a string to create a simple item with default configuration as shown for `Email` item. 
```

Comment:
I would move the last sentence to a separate paragraph. That way you'll have three paragraphs / "pieces of information":

- Paragraph 1: what's the entry point for configuration, what are the two types. 
- Paragraph 2: first type (simple string)
- Paragraph 3: second type (objects) 

Right now you describe two types, then list the first, and then the thought breaks off.